core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -7412,7 +7412,15 @@ public String toString() {
   public static final PropertyKey DORA_WORKER_METASTORE_ROCKSDB_DIR =
       stringBuilder(Name.DORA_WORKER_METASTORE_ROCKSDB_DIR)
           .setDefaultValue(format("${%s}/metastore", Name.WORK_DIR))
-          .setDescription("The base dir of RocksDB to store Dora meta data")
+          .setDescription("The base dir of RocksDB to store Dora metadata")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
+          .setScope(Scope.WORKER)
+          .build();
+
+  public static final PropertyKey DORA_WORKER_METASTORE_ROCKSDB_TTL =
+      intBuilder(Name.DORA_WORKER_METASTORE_ROCKSDB_TTL)
```

Comment:
Use durationBuilder? which supports input similar to `24h`

Reply:
not set or `-1` means no ttl?

Reply:
Fixed. Now use Duration to set TTL.
Zero or negative value means no TTL.